ros2_batch_job/__main__.py

```diff
@@ -76,6 +76,7 @@
     'pytest-repeat',
     'pytest-rerunfailures',
     'pytest-runner',
+    'pytest-timeout',
```

Comment:
We're generally preferring to use the system versions of dependencies on Linux now.  What this means is that you should add `python3-pytest-timeout` as an apt installed dependency to https://github.com/ros2/ci/blob/master/linux_docker_resources/Dockerfile , and as a dnf installed dependency to https://github.com/ros2/ci/blob/master/linux_docker_resources/Dockerfile-RHEL .  Then in this file you should follow the pattern in https://github.com/ros2/ci/blob/master/ros2_batch_job/__main__.py#L535-L540 and try to import `pytest_timeout`, and only install it via pip if that fails.

Reply:
Should be fixed in https://github.com/ros2/ci/pull/632/commits/54c6aafcfa866a79d0dc46d672205581fc169ca3.